alcs-frontend/src/app/features/application/decision/decision-v2/decision-documents/decision-documents.component.scss

```diff
@@ -22,3 +22,20 @@
 a {
   word-break: break-all;
 }
+
+table {
+  box-shadow: none;
+
+  th {
+    font-weight: 700;
+    padding-bottom: 16px;
+    position: relative;
+
+    .subheading {
```

Comment:
Sneaky using position absolute

Reply:
yep, otherwise table is going crazy 🙁

Reply:
That's the issue I had so I left it alone haha